lib/Connectable.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+interface Connectable
+{
+    /** 
```

Comment:
:no_entry_sign: **Error**: Missing short description in doc comment (*Generic.Commenting.DocComment.MissingShort*).